VeroliBGV/BGV009.xml

```diff
@@ -0,0 +1,1078 @@
+                           </item>
+                        </list>
+                     </additions>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate notBefore="1700" notAfter="1900" evidence="lettering">
+                           The handwriting is consistent with a 18th/19th-century dating.
+                           Several individuals are mentioned in the supplication formulas throughout the manuscript, following the phrase <foreign xml:lang="gez">ገብሩ፡</foreign>
+                           'his servant' or <foreign xml:lang="gez">ዓመቱ፡</foreign> 'his maid': 
+                           <persName role="patron" ref="PRS13330GadlaMi">Gadla Mikāʾel</persName>, his father <persName ref="PRS13331TaklaMi">Takla Mikāʾel</persName>, 
+                           his mother <persName ref="PRS13332Arwa">ʾArwā</persName>, his wife <persName ref="PRS13333WalattaMa">Walatta Māryām</persName>, their sons 
+                           <persName ref="PRS13334GabraAm">Gabra ʾAmlāk</persName>, <persName ref="PRS13335WaldaAm">Walda ʾAmlāk</persName>, 
+                           <persName ref="PRS13336SebhatLaab">Sǝbḥat Laʾab</persName>.
+                           <persName ref="PRS13337WaldaSa" role ="scribe">Walda Samāʿt</persName> (the second part of the name is effaced and thus hardly legible), indicated as the scribe on <locus target="#58va"/>.
+                           All these names are often effaced or erased, especially in the first half of the manuscript, but never replaced with other names. 
+                           Furthermore, <persName role="patron" ref="PRS13330GadlaMi">Gadla Mikāʾel</persName> is indicated as the patron in <ref target="#a1"/> and on 
+                           <locus target="#74va"/>.
+                        </origDate>
+                     </origin>                   
+                  </history>
+               </msPart>
+          
+                  <msPart xml:id="p2">
+                  <msIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <summary/>
+                     <msItem xml:id="p2_i1">
+                        <locus from="76ra" to="77ra"/>
+                        <title type="incomplete" ref="LIT3073RepCh353"/>
+                        <textLang xml:lang="gez"/>
+                        <note>The text is written in <ref target="#h2"/> and is the continuation of <ref target="p1_i3"/>, which begins on <locus target="#75ra"/>.
+                           The concluding supplication is seemingly incomplete.</note>
+                        <incipit xml:lang="gez">
+                           <locus target="#76ra"/>
+                           ልቀላ። አነ፡ ወአነ፡ <surplus>የ፡</surplus> <add place="interlinear">እቀድሞ፡</add> እንዘ፡ ይትበኃላ፡ <hi rend="rubric">ኤዎስጣቴዎስ<supplied reason="omitted" resp="MV">፡</supplied></hi>
```

Comment:
ልቀላ። = Now I see that it continues here. But I do not understand why we have to split it.

Reply:
This is a tricky issue, which I am not sure to have solved in the best way. In this MS a certain text (LIT3073RepCh353) begins in the last folios of the first codicological unit (p1) and continues on the second codicological unit (p2). This happened because another codicological unit (a simple bifolium) was added at the end of the original unit but the scribe decided, for several reasons (textual continuity, to save space), to start writing from the first available leaf, that is at the end of p1.
So, how to deal with these cases? I tried to duplicate the textual information, and I've described the same text both as `<msItem xml:id="p1_i3">` (last text of p1) and as `<msItem xml:id="p2_i1">` (first and only text of p2), both with the same ID = LIT3073RepCh353. The incipit of p1_i3 is the actual rubricated incipit, while the explicit of p2_i1 is the real conclusion of the text. But the explicit of p1_i2 just coincides arbitrarily with the end of the first codicological unit (in this case the word አንቀልቀላ። is split), and the incipit of p2_i1 similarly coincides with the first lines of the text at the beginning of the second codicological unit. I am wondering whether it is better to omit the last two pieces of information, which are potentially misleading for the reader (who might erroneously believe that the text is incomplete or acephalous).
More generally, I do not know if my solution is the ideal one for such cases. In most cases textual units are simple submultiples of codicological units, but here there is an overlap between the codicological and the textual sequences.
I am inclined to open an issue also for this and ask for Pietro's suggestion.

Reply:
I think that it largely depends on how we are going to visualize this information in the catalogue. I would avoid texts cut or incomplete, which do not give a proper idea of the complexity of this (interesting) phenomenon. Maybe, if you open an issue, you can already ask Pietro which are the possibilities of rendering this information in a clear way in our printed catalogue.
Many thanks, Massimo!